translations/Hindi/general-rules.md

```diff
@@ -0,0 +1,39 @@
+
+## Overview
+
+* Hindi is a very flexible & simple language. Since it is very much like English, the translations are an easy task.
+
+* Translations can be either formal or informal, but formal translations are preferred for corporate work.
+
+* The translations can be done in any person (first, second or third) according to the content one wants to translate.
+  <br> **Note:** For usage in zulip where we give instructions to the user, it would be recommended to use first person in formal. This will also maintain consistency with the other sites that the user may browse.
+
+* Similarly, the desicion to use the singular or plural forms also depends upon the type of content.
+
+* The usage of verbs & nouns is similar to English, the type of the verb & the noun used changes according to the construction of the sentence & the content, so there are no such rules for verbs & nouns.
+  <br> **Note:** In languages similar to Hindi, it is very common that the root word gets a suffix or is changed based on the sentence a lot more frequently than English. This is **NOT** the case with Hindi, therefore care has to be taken while translating that the word has been converted to the correct form properly.
+  <br> Check this link for [Hindi](https://translate.google.com/#en/hi/1.%20I%20will%20check%20this%20topic%20later.%0A2.%20I%20have%20no%20interest%20in%20this%20topic.%0A3.%20I%20want%20to%20talk%20on%20this%20topic.) and [Marathi](https://translate.google.com/#en/mr/1.%20I%20will%20check%20this%20topic%20later.%0A2.%20I%20have%20no%20interest%20in%20this%20topic.%0A3.%20I%20want%20to%20talk%20on%20this%20topic.) to see the difference.
+  <br> (Note the changes in the word *topic*, which translates to विषय in both the languages)
+
+* The usage of particles (e.g. reflexives) again depends upon the sentence, so their is no restriction on their usage.
+
+* Slang words (especially abusive words) should avoided while translating, and it would be highly recommended to not use them in a formal environment like Zulip
+
+* Long words are very uncommon in Hindi (especially Zulip-specific), so the translations are easy & quick.
+
+* There is no specific way of addressing people in Hindi, it can be done in any way (formal or informal), but formal translations are suggested for Zulip. (Also, personal pronouns can be used freely, there is no restriction or convention.) 
+  <br>An example would we could use here is translation of *Send me a message*
+  1. Formal: 'कृपया मुझे एक संदेश भेजें'
+  2. Informal: 'मुझे एक संदेश भेजो'
+
+* Capitalization is not done in Hindi in any form, as there are no uppercase & lowercase in the language.
+  
+* Other grammatical elements (such as gerunds) are used again according to the content, similar to how it is done in English.
+
+* Translation of Zulip-specific terms & common verbs is also very easy in Hindi.
+  e.g,
+  1. The word *message* translates to 'संदेश' in Hindi.
+  2. The verb *check* translates to 'चेक' in Hindi.
```

Comment:
It doesn't. The translation of 'Please check the PR' below looks correct. This, on the other hand, is simply the way to write 'check' in Devanagiri Script.

Reply:
Will update it, but it is accepted as an uptaken word as far as I know

Reply:
Used a different word there